Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -243,4 +256,53 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
 
     puts "No unbottled dependencies found!"
   end
+
+  def output_lost_bottles
+    # $ git log --patch -G\^\ \+sha256.\*\ sonoma: --since=@\{\'1\ week\ ago\'\}
+    git_log = %w[git log --patch]
```

Comment:
Would using `FormulaVersions` make sense here at all?

Reply:
I don't think so. Do you have something specific in mind?

Reply:
@apainintheneck My thinking was that `FormulaVersion`s also iterates through `git` commits looking for things in the formula DSL so that it could be a pretty good fit here, particularly as it's actually reading the `Formula` rather than using regex.

I think it's worth a (brief, time boxed) investigation but utterly defer to you if you feel it's a bad fit and doesn't need to block this PR either way 👍🏻

Reply:
I'm not super familiar with `FormulaVersions` but it looks like it operates on a formula-by-formula basis. That means that using it here would require both loading all formulae and then making one Git call per formula.

In comparison, this approach doesn't need to load any formulae and only makes one call out to Git which I think is more ideal.

Reply:
@apainintheneck Cool, all good!